mods/AdvancedItem/assets/advanceditem/recipes/grid/wand.json

```diff
@@ -0,0 +1,6 @@
+{
+	"ingredientPattern": "P,S,S",
+	"ingredients": {
+		
+	}
```

Comment:
seems like you're missing the ingredients, is this intended ?

Reply:
yeah that mod was only started, i probably should not have committed it.